aws_xray_sdk/core/models/mimic_segment.py

```diff
@@ -0,0 +1,35 @@
+from .segment import Segment
+from ..exceptions.exceptions import MimicSegmentInvalidException
+
+
+class MimicSegment(Segment):
+    """
+    The MimicSegment is an entity that mimics a segment for the use of the serverless context.
```

Comment:
`serverless context` is too generic. This is a special workaround with lambda due to lambda integration implementation details. We should be clear on the intention of this "hacky" class type.

Reply:
Done